src/webui/web_hooks.py

```diff
@@ -71,11 +71,11 @@ async def github_webhook(
         return {'message': 'Event not supported'}
 
     if event_type == 'issues':
-        logfire.info('Received GitHub issue event: {data}', data=data)
+        logfire.info('Received GitHub issue event: {gh_data}', gh_data=data)
```

Comment:
I renamed these so that I could whitelist `['attributes', 'gh_data']` instead of `['attributes', 'data']` which would be too generic.